Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/DiagnosticsContainer.cs

```diff
@@ -19,11 +19,12 @@ public void Dispose()
             Debug.unityLogger.logHandler = defaultLogHandler;
         }
 
-        public static DiagnosticsContainer Create(IReportsHandlingSettings settings, params (ReportHandler, IReportHandler)[] additionalHandlers)
+        public static DiagnosticsContainer Create(IReportsHandlingSettings settings, bool enableLocalSceneReporting = false, params (ReportHandler, IReportHandler)[] additionalHandlers)
         {
             settings.NotifyErrorDebugLogDisabled();
 
-            List<(ReportHandler, IReportHandler)> handlers = new List<(ReportHandler, IReportHandler)>(additionalHandlers.Length + 2);
+            int handlersCount = additionalHandlers.Length + 2 + (enableLocalSceneReporting ? 1 : 0);
```

Comment:
Lets extract "+ 2 +" to a const value to give more context with a first look on the code

Reply:
implemented at https://github.com/decentraland/unity-explorer/pull/1546/commits/0d20de12ee24b27c5ff92c7bcffe190a680f4cd7